src/Aquifer.API/Endpoints/Help/Aquifer-cms/Documents/Response.cs

```diff
@@ -0,0 +1,17 @@
+using Aquifer.Data.Entities;
+
+namespace Aquifer.API.Endpoints.Help.Aquifer_cms.Documents;
+
+public class Response
+{
+    public required List<HelpDocumentResponse> Releases { get; set; } = [];
+    public required List<HelpDocumentResponse> HowTos { get; set; } = [];
```

Comment:
```suggestion
    public required List<HelpDocumentResponse> Releases { get; init; }
    public required List<HelpDocumentResponse> HowTos { get; init; }
```
You shouldn't need the default empty collection assignment if you switch to `init` instead of `set` which allows an object initializer (rather than only the constructor) to set values.

Reply:
Good insight. Implemented. TY.